ports/getopt-win32/vcpkg.json

```
@@ -1,9 +1,9 @@
 {
   "name": "getopt-win32",
   "version-string": "0.1",
```

```suggestion
  "version": "0.1",
```

Shouldn't this be version?

---

Fixed in new commit